ui-v2/app/styles/components/healthcheck-output/skin.scss

```diff
@@ -14,6 +14,13 @@
     margin-right: 8px;
   }
 }
+%healthcheck-output dd em {
+  @extend %pill;
+  cursor: default;
+  font-style: normal;
```

Comment:
Why do these top two attributes need to be added here, but not in `%pill`? The bottom two make sense to me based on this location.

Reply:
Definitely, they probably should be, but I don't want to change too much in this PR, especially with CSS changes and especially things in the `base` CSS . If anything it could do with a TODO maybe. I'll add one here.